franklin/config/urls.py

```diff
@@ -18,7 +18,6 @@
 
 urlpatterns = [
     url(r'^', include('core.urls')),
-    url(r'^docs/', include('rest_framework_swagger.urls')),
```

Comment:
Swagger was more trouble than it's worth. There was a story to remove it. Several recent changes "broke" the existing docs, so I decided now was a good time to rip this out since the docs are wrong anyway.